### DISCUSSION TITLE
[已解决]REALITY系列组合如 H2 gRPC 的模板（命名）格式？

### DISCUSSION THREAD

**Discussion by chika0801** [XTLS/Xray-core#1720] [Q&A]
如果参考 VLESS-XTLS-uTLS-REALITY

那么 VLESS-H2-uTLS-REALITY

VLESS-gRPC-uTLS-REALITY

按这样的规律命名可好了？

**Comment by RPRX** [comment]
~~VLESS-WS-uTLS-REALITY~~
